manifests/env/nostromo-stage/telegraf/telegraf.conf

```diff
@@ -102,13 +86,12 @@ tags = ["CurrentLap", "CarClass"]
 
 [[outputs.influxdb_v2]]
 bucket = "racing"
+namepass = ["laps*"]
 organization = "b4mad"
 token = "$INFLUX_TOKEN"
 urls = ["http://influxdb2:8086"]
-namepass = ["laps*"]
```

Comment:
and without this, we put the metrics meant for prometheus into influx

Reply:
pls recheck e78a167